### PR TITLE
Allow a notifying organisation to enter variation details

### PR DESCRIPTION
**Pull request by ben-cart3r**
- Add a simple page to enter details about a variation (date and type)

### CODE REVIEW

File: server/models/view-models/variationDetails.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      month: '',
+      year: '',
+    }
+  }
+
+  const date = new Date(dateString)
+
+  return {
+    day: date.getDate().toString(),
+    month: (date.getMonth() + 1).toString(),
```
Comment (CslBing) [inline server/models/view-models/variationDetails.ts:19]: why +1?

**Review by CslBing — APPROVED** [review]
👍

File: server/models/view-models/variationDetails.ts (hunk 1 above)
Comment (ben-cart3r) [reply server/models/view-models/variationDetails.ts:19]: Months are zero indexed

Comment (CslBing) [reply server/models/view-models/variationDetails.ts:19]: Oh, it's one of those using zero indexing thing 🤦

**Pull request merged**